CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changelog
-## Unreleased
+## v0.11.1
 
 ### Changed
 - [[#178](https://github.com/rust-vmm/vmm-sys-util/issues/178)]: Fixed a bug in
```

Comment:
This should probably be in a `Fixed` section, looks like we used to add all the fixes in the Changed section. Maybe in the future we can fix this, but for now let's merge it as is.